DVTShowcase/HomeViewController.swift

```diff
@@ -73,37 +85,15 @@ class HomeViewController: DVTShowcaseViewController, UITableViewDelegate, UITabl
         return cell;
     }
     
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        print("indexPathSelected: \(indexPath.row)")
```

Comment:
please remove this. dont keep prints in your code, only use for testing and remove it.